dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -158,8 +158,14 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
     inline std::wstring GetExeNameFromPath(std::wstring const& exePath)
```

Comment:
When were we sending a relative path as opposed to absolute path? Just curious :)

Reply:
For unpackaged, we send absolute path. For packaged, it's assigned in the manifest which for MSIX existed in a subfolder, but for Self-contained it was in the same BuildOutput.